force-app/main/default/classes/GE_PaymentServices.cls

```diff
@@ -153,8 +153,21 @@ public with sharing class GE_PaymentServices {
      * @return String JSON of visualforce URL
      */
     @AuraEnabled(cacheable=true)
-    public static String getVfURL(String namespace) {
-        return DomainCreator.getVisualforceHostname(namespace);
+    public static OriginUrls getOriginUrls(String namespace) {
+        return new OriginUrls(
+            DomainCreator.getVisualforceHostname(namespace),
+                DomainCreator.getLightningHostname()
+        );
+    }
+
+    public class OriginUrls {
+        @AuraEnabled public String visualForceOriginUrl;
+        @AuraEnabled public String lightningOriginUrl;
+
+        public OriginUrls(String visualForceOriginUrl, String lightningOriginUrl) {
```

Comment:
This looks good to me, I like that it's an object of URLs instead of two separate functions, I honestly wish we could use this for the `buildLightningOrigin` lightningURL in `GE_tokenizecard` instead of having a separate method but I believe that can't be done if something is Auraenabled, correct? One thing to note (not a change needed to be made here of course) but if we want to return the url with the `https://` from the backend side for the sake of consistency  we should do the same for this method as well 
``` public String getLightningURL() {
        return DomainCreator.getLightningHostname();
    }
```